tests/unit/test_hyper_transformer.py

```diff
@@ -325,9 +322,8 @@ def test__get_next_transformer_next_transformers(self):
         """
         # Setup
         transformer = FrequencyEncoder()
-        ht = HyperTransformer(
-            default_sdtype_transformers={'categorical': OneHotEncoder()}
-        )
+        ht = HyperTransformer()
+        ht.default_sdtype_transformers = {'categorical': OneHotEncoder()}
```

Comment:
I think we might want to check the whole codebase for this and make sure it is private now